endpoints/openrtb2/amp_auction_test.go

```diff
@@ -1028,40 +1028,50 @@ func TestAMPSiteExt(t *testing.T) {
 
 // TestBadRequests makes sure we return 400's on bad requests.
 func TestAmpBadRequests(t *testing.T) {
-	dir := "sample-requests/invalid-whole"
+	dir := "sample-requests/amp/invalid-whole"
```

Comment:
This is a pretty significant change. We're now executing just a single test rather than the 75 or so tests that reside in `sample-requests/invalid-whole`. What's the motivation here behind this change?

Reply:
reverted this change, will raise a separate PR to discuss the issue with `TestAmpBadRequests()`. It does not correctly validate the expected failure similar to how invalid cases are validated by UT of /auction endpoint.